src/small-component.scss

```diff
@@ -4,6 +4,142 @@
  * Use this style sheet to define style for small component
  */
 
+
+.gc-form {
+	clear: both;
+	padding-top: 6px;
+}
+.gc-form {
+	.checkbox, .radio, .form-checkbox, .form-radio {
+		-moz-user-select: none;
```

Comment:
There is no need to include any browser vendor variant, the SASS compiler will do it automatically

Reply:
I'll remove those then, thanks.